### PR TITLE
added notification reminders before 24 hrs and before 1hr of the scheduled call

### PR DESCRIPTION
**Pull request by ChaitanyaV98**

### CODE REVIEW

File: app/functions/100ms/notificationReminder/index.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+      let buyerProfile = await Profile.findOne({
+        where: {
+          uuid: consultationRequest.buyer_uuid,
+          type: "fm-buyer",
+        },
+      });
+
+      // sending notification to banker and buyer before 24 hr of the consultation
+      if (timeForConsultationInHours === +24 && minutesDiff === +1440) {
```
Comment (rajeshn95) [inline app/functions/100ms/notificationReminder/index.js:71]: @ChaitanyaV98 what is +24 here?

File: app/functions/100ms/notificationReminder/index.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+            {
+              type: "email",
+              value: buyerProfile.data.email,
+            },
+          ],
+        });
+      }
+
+      // sending notification to banker and buyer before 1 hr of the consultation
+      else if (timeForConsultationInHours === +1 && minutesDiff === +60) {
```
Comment (rajeshn95) [inline app/functions/100ms/notificationReminder/index.js:117]: @ChaitanyaV98 what is +1 here?

**Review by rajeshn95 — CHANGES_REQUESTED** [review]

File: app/functions/100ms/notificationReminder/index.js (hunk 2 above)
Comment (ChaitanyaV98) [reply app/functions/100ms/notificationReminder/index.js:117]: actually `timeForConsultationInHours` and `minutesDiff` are of number type, so thats why +1, +60 etc.. which will be considered as number data type.

File: app/functions/100ms/notificationReminder/index.js (hunk 1 above)
Comment (ChaitanyaV98) [reply app/functions/100ms/notificationReminder/index.js:71]: actually `timeForConsultationInHours` and `minutesDiff` are of number type, so thats why +1, +60 etc.. which will be considered as number data type, +24 is the number of hours.!

**Pull request merged**